README.md

```diff
@@ -72,6 +72,37 @@ asyncio.run(client.subscribe(query=query, handle=print))
 # ...
 ```
 
+## Advanced Usage
+
+### Disable SSL verification
+
+Set `options` to `{"verify": False}` ether when instantiating the `GraphqlClient` class.
+
+```py
+from python_graphql_client import GraphqlClient
+
+client = GraphqlClient(endpoint="wss://www.your-api.com/graphql", verify=False)
+```
+
+Alternatively, you can set it when calling the `execute` method.
+
+```py
+from python_graphql_client import GraphqlClient
+
+client = GraphqlClient(endpoint="wss://www.your-api.com/graphql"
+client.execute(query="<Your Query>", verify=False))
+```
+
+### Custom Authentication
+
+```py
```

Comment:
TIL `py` can be used as shorthand for `python` for markdown code blocks 🤯

Reply:
haha. I've also updated the other places to use `py` instead of `python` ~to save us a little bit of typing~ for consistency.